corehq/apps/userreports/reports/sum_when_templates.py

```diff
@@ -64,6 +64,11 @@ class CCSPhaseTemplateSpec(SumWhenTemplateSpec):
     expression = "ccs_phase = ?"
 
 
+class ComplementaryFeedingTemplateSpec(SumWhenTemplateSpec):
+    type = TypeProperty("complementary_feeding")
+    expression = "is_cf = ?"
```

Comment:
Is this suppose to be a boolean expression now or is_cf == 'cf'?

Reply:
is_cf is essentially a boolean, but its value is either "cf" or "non_cf": https://github.com/dimagi/commcare-hq/blob/20a47ee1c129714bc0fb87157926c0fec15b19c1/custom/icds_reports/ucr/data_sources/child_cases_monthly_v2.json#L1858-L1865

Reply:
:(